ios/FluentUI/Other Cells/TableViewCellFileAccessoryView.swift

```diff
@@ -548,4 +551,15 @@ extension UIContentSizeCategory {
             return true
         }
     }
+
+    var dateLabelNumberOfLines: Int {
+        switch self {
+        case .accessibilityExtraExtraExtraLarge, .accessibilityExtraExtraLarge, .accessibilityExtraLarge:
+            // Three lines for accessibility XXXL/XXL/XL content size categories (like the title label) so that
+            // the text can fit in its label's available width without overlapping with accessory buttons.
+            return 3
```

Comment:
should we really limit to three? might be better if we cap the font size? or adjust font one width to certain point?

Reply:
I think it makes sense to keep the same font size as the other labels in the text area are using.
I also thought about setting that to unlimited, but it could potentially grow bigger in different locales.
This is the reasoning behind keeping it in sync with the title label logic of 3 lines.

Reply:
On a related note, is there any need for a line limit at all? What would happen if we just always set the limit to 0?

Reply:
If in a certain locale the string needs more than 3 lines, the height calculation might be off. Also, this is the logic for the title label, which is more important information than the date label and is still capped at 3 lines.

Reply:
does it already support large content viewer ?

Reply:
Currently it doesn't support the large content viewer although the content increases in size along with the other labels.
We can follow up on our offline conversation about supporting it in case the text truncates and make an additional tweak if needed.